training/nvidia-bootc/Containerfile

```diff
@@ -182,13 +182,10 @@ fi
 # Also make sure not to duplicate if a base image already has it specified.
 RUN grep -q /usr/lib/containers/storage /etc/containers/storage.conf || \
     sed -i -e '/additionalimage.*/a "/usr/lib/containers/storage",' \
-	/etc/containers/storage.conf && \
-    if [ -f "/run/.input/ilab" ]; then \
-	  cp /run/.input/ilab /usr/bin/ilab; \
-     else \
-	  curl -o /usr/bin/ilab "https://raw.githubusercontent.com/containers/ai-lab-recipes/main/training/ilab-wrapper/ilab"; \
```

Comment:
I though it was better to always pick the local version, to have build reproducibility